apps/src/templates/sectionProgress/LessonSelector.jsx

```diff
@@ -8,8 +8,7 @@ const styles = {
     display: 'block',
     boxSizing: 'border-box',
     fontSize: 'medium',
-    padding: '0.8em',
-    marginBottom: 10,
+    height: 34
   },
   heading: {
     marginBottom: 0,
```

Comment:
Question:  Why is there a comma after the marginBottom (it's the only property:value pair for heading)?  There isn't a comma after (height: 34), which is inline with the style guide.

Reply:
Oh, the comma is on line 15 is probably because I had a style after marginBottom that I ended up not needing so I removed it. I have a slight preference for trailing commas because if I add something later, I will only see the change that I added in the diff not the changes that I added AND the additional comma on the line preceding the change. 🤷🏼‍♀️

Reply:
Thanks